leaflet-providers.js

```diff
@@ -175,7 +175,7 @@
 		},
 		MapBox: {
 			url: function (id) {
-				return 'http://{s}.tiles.mapbox.com/v3/' + id + '/{z}/{x}/{y}.png';
+				return '//{s}.tiles.mapbox.com/v3/' + id + '/{z}/{x}/{y}.png';
```

Comment:
Yeah, but should be fixed by this:
https://github.com/leaflet-extras/leaflet-providers/blob/mapbox-adjustment/leaflet-providers.js#L44-L47